azurerm/internal/services/keyvault/key_vault_key_decrypt_data_source.go

```diff
@@ -0,0 +1,83 @@
+package keyvault
```

Comment:
If the intention is to decrypt secrets, might i suggest that the resource instead be called:
`azurerm_key_vault_secret_decrypt`

Reply:
as above